docs/meetings/team/2023_02_27.md

```diff
@@ -0,0 +1,74 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:06
+
+**End Time** 10:36
+
+**Location** Discord
+
+**Date** February 27, 2023
+
+## 👋 Attendence
+
+1. Veronica
+2. Matt
+3. Logan
+
+***Attendence Comments***
+
+- Logan is in Hawaii
+- Ed is traveling back from Japan
+
+## 🃏 Joke of the Day
+
+Did you know that there's a crocodile that can jump higher than the average house?
+This is due to its powerful back legs and the fact that the average house can't jump!
+
+## 📢 Announcements
+
+- Welcome back from Reading Week!
+- Due this week:
+  - GitStats Week 21 due this Friday
+  - Usability Task List
+  - Video Demo
+- Next Week
+  - Peer Testing II activities
+
+## 📅 Last 2 Weeks (Restrospective)
+
+- We're nearing the end which feels good, and we feel good about our progress
+- Veronica updated the usability tasks, but we should go over it together as a group!
+
+## 📋 This Week
+
+### Assets
+
+- Current temporary poses
+  - Will need to reach out to client so that we can receive official poses from artist
+  - Once recieved, we'll need to export the assets for our file setup
+- Will we want transition poses or something for the transition to a new pose? In the YouTube demo, the stick figure
+bounced a bit when it went into a new pose.
+  - Due to the timings of our frames, implementing the transitions may prove difficult or look strange
+  - Let's see if it's mentioned in peer testing
+
+### Issues this week
+
+- We want to figure out why the servers are performing worse than our local machines (Matt)
+  - Need to investigate if our API is opening threads appropriately
+- Update Usability Task List (Veronica)
+  - repeat scenarios only if there's time
+  - test with server rather than local machine
+  - update instructions for peer testing to fill out Google Form at the same time
+- Google Form for feedback (Team)
+  - keep track of parameters for each scenario
+  - list of questions for feedback
+- Review current documentation to ensure it's accurate and updated, then convert to GitHub Pages
+  - Currently, we have our README, separate API documentation, and separate tooling documentation
+- Video demo: let's screen record and voice over again.
+  - It's a busy week, so it might only be done with a couple group members instead of everyone
+
+## 🤝 Client Meeting
+
+- Show off poses and high-resolution
+- Check on the asset pose request
+- Performance report: Show our results in a table for audio files of different lengths (10s, 30s, 1:00, 1:30, etc.)
```

Comment:
if it seems appropriate we should discuss the importance of performance on their system. What are the next steps for us ?

Reply:
This is definitely something we should bring up with them in the next meeting!